### ISSUE TITLE
Missing TEMPLATE_DIRS in README example

### ISSUE THREAD

**Issue by jbbarth** [zenwalker/django-micro#6]
Just did a copy/paste of the README, and go this:
```
% python app.py runserver
Traceback (most recent call last):
  File "app.py", line 5, in <module>
    configure(locals())
  File "/Users/jbbarth/.virtualenvs/elastictac/lib/python2.7/site-packages/django_micro.py", line 81, in configure
    'DIRS': config_dict.pop('TEMPLATE_DIRS'),
KeyError: 'TEMPLATE_DIRS'
```

Having a `TEMPLATE_DIRS = []` above the `configure(locals())` call seems to fix it. I'm running django-micro-1.4.1 with django-1.10.7 at the moment.

Thanks for your work anyway, I'm giving it a try today and I already like it :-)

**Comment by jbbarth** [comment]
Ah! Just saw TEMPLATE_DIRS is mentioned below in the README. It's just missing in the first example. I can provide PRs as I'm discovering the lib if you want.

**Comment by maxpoletaev** [comment]
This setting is not required. By default Django discover templates in `templates` directory inside your app. Sorry for this mistake. Fix will be released in pypi as 1.4.2 next few hours.
